### PR TITLE
simdjson: update to 3.3.0

### PR DESCRIPTION
**Pull request by shanoaice**
edit: closes #930 too

### CODE REVIEW

**Comment by neheb** [comment]
Need releases.json entry

**Comment by neheb** [comment]
could you integrate and fix https://github.com/mesonbuild/wrapdb/pull/930 here?

**Comment by shanoaice** [comment]
> could you integrate and fix #930 here?

Will try

File: subprojects/simdjson.wrap [hunk 1]
```diff
@@ -1,8 +1,8 @@
 [wrap-file]
-directory = simdjson-3.1.1
-source_url = https://github.com/simdjson/simdjson/archive/refs/tags/v3.1.1.tar.gz
-source_filename = simdjson-3.1.1.tar.gz
-source_hash = 4fcb1c9b1944e2eb8a4a4a22c979e2827165216f859e94d93c846c1261e0e432
+directory = simdjson-3.3.0
+source_url = https://github.com/simdjson/simdjson/archive/refs/tags/v3.3.0.tar.gz
+source_filename = simdjson-3.3.0.tar.gz
+source_hash = a8c9feff2f19c3ff281d42f0b6b4b18f02236513b99229756fa9a1b14787a58a
 patch_directory = simdjson
```
Comment (neheb) [inline subprojects/simdjson.wrap:6]: need to edit the meson files...

**Comment by shanoaice** [comment]
> could you integrate and fix #930 here?

Done, tested on local machine and it seems to work fine.

**Comment by shanoaice** [comment]
This failure is odd...should I squash both changes into one commit and retry?

**Comment by shanoaice** [comment]
> Reorder your versions in `releases.json`: last release first.

Oh thanks, I think now it's good.

**Comment by neheb** [comment]
Add

```
gnu_symbol_visibility: 'hidden'
```

to the library

**Pull request merged**